### ISSUE TITLE
About how the DAIC_WOZ_process framework is used in the DepAudioNet_reproduction project file, and some confusing file generation causing memory explosions

### ISSUE THREAD

**Issue by Jin-Tao-208** [adbailey1/DepAudioNet_reproduction#8]
Hello Andrew, 
I saw in the `README` that you mentioned needing to use the `DAIC_WOZ_process` framework to preprocess audio features. So my feature folder `mel_svn_exp`, which I generated using the` DAIC_WOZ_process` framework, is placed in the project folder `DepAudioNet_reproduction`, which has the following file structure. 
**/media/disk/DepAudioNet**$ ls -h
**DAIC-WOZ**  &ensp;   **daic_woz_audio**   &ensp;   **daic_woz_text**   &ensp;  **data_loader**  &ensp;    **exp_run**          **mel_svn_exp**  &ensp; **utilities**      &ensp;     README.md        &ensp;     **daic_woz_config_files**     &ensp;        **daic_woz_unit_test**     &ensp;      environment.txt  &ensp;    run.shcondor      &ensp;     **daic_woz_run**    &ensp;           **daic_woz_utils**       &ensp;      environment.yml    &ensp;         main1.py         &ensp;    **unit_test**
 
But oddly, when I was training the data set( `python3 main1.py train --validate --cuda --vis --position=1`), I had a loop-nesting situation generating the mel_SVN_exp folder under the mel_SVN_exp folder. The generated file structure is as follows:`/media/disk/DepAudioNet/mel_svn_exp/exp_1/daic/mel_svn_exp/exp_1/daic/mel_svn_exp/exp_1/daic/mel_svn_exp`

This caused my hard drive to explode incapacity, at which point the System ran and said `"System will exit as the validation loss has not improved for {patience} epochs".`  I am very confused as to why this situation can happen. Hope to answer.

Looking forward to your reply.
Thank you very much.

Best
Stella

**Comment by adbailey1** [comment]
Hi Stella, 

This is strange and I am not sure why this is happening but if I share an example of my file structure and execution logic, hopefully it will help.

Features stored at: /media/daic_woz/raw_svn_exp (this has complete_database.h5)
My code is at: /home/DepAudioNet_reproduction

Therefore in config:

WORKSPACE_MAIN_DIR = '/media/daic_woz'
WORKSPACE_FILES_DIR = os.path.join('/home', 'DepAudioNet_reproduction')

In main1.py line 1473:
features_dir = '/media/daic_woz/raw_svn_exp'

In main1.py line 1492:
current_dir = '/media/daic_woz/raw_svn_exp/exp_1'

in main1.py line 1496:
model_dir = '/media/daic_woz/raw_svn_exp/exp_1/model/1'

Do you have a similar output to me? If not then we need to try looking at something else. 

Kind regards, 

Andrew

**Comment by Jin-Tao-208** [comment]
Hi Andrew,

I ran the program again according to your file structure. There will be no such problems. Thanks for your help! It's so kind of you!

Best
Stella

**Comment by adbailey1** [comment]
My pleasure, glad it worked! I shall make a note to update the variable names to make things clearer!

Kind regards, 

Andrew
